resources/qml/SidebarSimple.qml

```diff
@@ -270,46 +254,61 @@ Item
         }
 
         Label{
-            id: supportHelperLabel
+            id: supportExtruderLabel
+            visible: (supportEnabled.properties.value == "True") && (machineExtruderCount.properties.value > 1)
             anchors.left: parent.left
             anchors.leftMargin: UM.Theme.getSize("default_margin").width
-            anchors.verticalCenter: supportCheckBox.verticalCenter
+            anchors.verticalCenter: supportExtruderCombobox.verticalCenter
             width: parent.width / 100 * 35 - 3 * UM.Theme.getSize("default_margin").width
-            //: Support label
-            text: "";
+            text: catalog.i18nc("@label", "Support Extruder");
             font: UM.Theme.getFont("default");
             color: UM.Theme.getColor("text");
         }
 
-        CheckBox{
-            id: supportCheckBox
-            visible: machineExtruderCount.properties.value <= 1
-            property alias _hovered: supportMouseArea.containsMouse
+        ComboBox {
+            id: supportExtruderCombobox
+            visible: (supportEnabled.properties.value == "True") && (machineExtruderCount.properties.value > 1)
+            model: extruderModel
 
-            anchors.top: adhesionCheckBox.bottom
-            anchors.topMargin: UM.Theme.getSize("default_margin").height
-            anchors.left: supportHelperLabel.right
+            anchors.top: enableSupportCheckBox.bottom
+            anchors.topMargin: {
+                if ((supportEnabled.properties.value == "True") && (machineExtruderCount.properties.value > 1)) {
+                    return UM.Theme.getSize("default_margin").height;
+                } else {
+                    return 0;
+                }
+            }
+            anchors.left: supportExtruderLabel.right
             anchors.leftMargin: UM.Theme.getSize("default_margin").width
+            width: parent.width / 100 * 65
+            height: {
+                if ((supportEnabled.properties.value == "True") && (machineExtruderCount.properties.value > 1)) {
+                    // default height when control is enabled
+                    return UM.Theme.getSize("setting_control").height;
+                } else {
+                    return 0;
+                }
+            }
 
-            //: Setting enable support checkbox
-            text: catalog.i18nc("@option:check", "Print Support Structure");
-            style: UM.Theme.styles.checkbox;
+            style: UM.Theme.styles.combobox
             enabled: base.settingsEnabled
+            property alias _hovered: supportExtruderMouseArea.containsMouse
 
-            checked: supportEnabled.properties.value == "True"
+            currentIndex: parseFloat(supportExtruderNr.properties.value)  // supportEnabled.properties.value == "True" ? parseFloat(supportExtruderNr.properties.value) : -1
```

Comment:
The code in the comment here is wrong as well as commented out. I removed it for you in 79e8fc210a8a3e20e256f232cdcff2470ca5350c.